frontend/src/metabase/admin/settings/components/SettingsPages/GeneralSettingsPage.tsx

```diff
@@ -0,0 +1,75 @@
+import { jt, t } from "ttag";
+
+import { useDocsUrl } from "metabase/common/hooks";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { Stack } from "metabase/ui";
+
+import { AdminSettingInput } from "../widgets/AdminSettingInput";
+import { AnonymousTrackingInput } from "../widgets/AnonymousTrackingInput";
+import { CustomHomepageDashboardSetting } from "../widgets/CustomHomepageDashboardSetting";
+import { HttpsOnlyWidget } from "../widgets/HttpsOnlyWidget";
+import { SiteUrlWidget } from "../widgets/SiteUrlWidget";
+
+export function GeneralSettingsPage() {
+  const { url: iframeDocsUrl } = useDocsUrl("configuring-metabase/settings", {
+    anchor: "allowed-domains-for-iframes-in-dashboards",
+  });
+
+  return (
+    <Stack gap="xl" maw="42rem" px="lg" py="sm">
+      <AdminSettingInput
+        name="site-name"
+        title={t`Site Name`}
+        description={t`The name used for this instance of Metabase.`}
+        inputType="text"
+      />
+
+      <SiteUrlWidget />
+
+      <HttpsOnlyWidget />
+
+      <CustomHomepageDashboardSetting />
+
```

Comment:
hey doofus, you forgot the landing page setting, which is an enterprise plugin:

`enterprise/frontend/src/metabase-enterprise/whitelabel/components/LandingPageWidget/LandingPageWidget.tsx`